site/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>amb</title>
+  <link href="../assets/styles/normalize.css" rel="stylesheet" >
+  <link href="../assets/styles/main.css" rel="stylesheet" >
+</head>
+
+<body>
+  <header>
+    <a href="index.html">
+      <div>
+        <h3>amb</h3>
+      </div>
+    </a>
+
+    <nav>
+      <ul>
+        <li><a href="portfolio.html"> porotfolio </a></li>
+        <li><a href="blog.html"> blog </a> </li>
+        <li><a href="about.html"> about </a></li>
+        <li><a href="#contact"> contact </a> </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section id = "index_hero" class = "hero">
+      <h1>amb</h1>
+    </section>
+
+    <section class = "index_section1 section1">
```

Comment:
index_section1 section1 seems redundant? If you need them both thinking about naming specificity.